sdk/src/context-core.ts

```diff
@@ -1,10 +1,10 @@
 import {GRAPHQL_NODES, IPFS_NODES, LIVE_CONTRACTS} from './constants';
```

Comment:
rethink file name

Reply:
is going to get deleted, this will be legacy code with the new sdk